### PR TITLE
Pow dynamic validator

### PR DESCRIPTION
**Pull request by limebell**

### CODE REVIEW

File: Libplanet/Consensus/Validator.cs [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            {
+                return Equals(other);
+            }
+
+            return false;
+        }
+
+        public bool Equals(Validator? other)
+        {
+            return PublicKey.Equals(other?.PublicKey);
```
Comment (greymistcube) [inline Libplanet/Consensus/Validator.cs:130]: Is not considering power intended? If so, why?

Comment (OnedgeLee) [reply Libplanet/Consensus/Validator.cs:130]: What I've thought was, we're often use validator's identity as it's public key.
In practical use, when we compare validator `A` and validator `B`, we often don't pay attention to its powers. What we're interested in is their identity.
So, I've thought judging `Validator("A", 100)` same as `Validator("A", 1000)` would be better for practical use.
I thought, when we're interested in their power, we would call `ValidatorA.Power.Equals(ValidatorB.Power)`.
May people can confused with this notation?

**Review by OnedgeLee — DISMISSED** [review]

File: Libplanet/Action/Sys/SetValidator.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        public SetValidator(PublicKey validatorPublicKey, BigInteger power)
+        {
+            ValidatorPublicKey = validatorPublicKey;
+            Power = power;
+        }
+
+        public SetValidator(Validator validator)
+            : this(validator.PublicKey, validator.Power)
+        {
+        }
```
Comment (greymistcube) [inline Libplanet/Action/Sys/SetValidator.cs:31]: Seems a bit redundant. I think `SetValidator(Validator)` should suffice?

Comment (limebell) [reply Libplanet/Action/Sys/SetValidator.cs:31]: For now, yes but in some cases such as mutation in headless, `SetValidator(PublicKey validatorPublicKey, BigInteger power)` can be used.

File: Libplanet/Consensus/Validator.cs (hunk 1 above)
Comment (OnedgeLee) [reply Libplanet/Consensus/Validator.cs:134]: Resolved

File: CHANGES.md [hunk 3]
```diff
@@ -56,9 +66,20 @@ To be released.
...
+ -  Added `ValidatorSetStateCompleter<T>` delegate.  [[#2716]]
+ -  Added `ValidatorSetStateCompleters<T>` class.  [[#2716]]
+ -  (Libplanet.Explorer) Added `ValidatorType` class.  [[#2716]]
+ -  (Libplanet.Explorer) Added `validators` query in `StateQuery`.  [[#2716]]
 
 ### Behavioral changes
 
+ -  `Swarm<T>` became to execute actions one by one when preloading blocks.
+    (used to execute actions after appending all blocks)  [[#2716]]
+
```
Comment (greymistcube) [inline CHANGES.md:82]: This no longer applies...? 😶

**Review by OnedgeLee — APPROVED** [review]

**Review by greymistcube — APPROVED** [review]

**Pull request merged**